packages/myst-cli/src/process/file.ts

```diff
@@ -43,7 +54,7 @@ export async function loadFile(
         const mdast = parseMyst(session, content, file);
         cache.$mdast[file] = {
           sha256,
-          pre: { kind: SourceFileKind.Article, file, mdast },
+          pre: { kind: SourceFileKind.Article, file, location, mdast },
```

Comment:
This probably makes sense to add here... Previously, we've avoided adding project-specific info to the file "pre" mdast content; the loaded file was independent of context. I don't really think that's important, though - the only case where this gets dicey is if multiple projects share a single file.


If we _don't_ compute the location here, we are stuck having to compute location when writing nested json later which is way messier I think.